src/main/java/uk/co/ramp/covid/simulation/Model.java

```diff
@@ -223,8 +222,8 @@ private void writeOutput(int iterId, List<List<DailyStats>> s) {
         }
         
         // By here the output directory will be available
-        outputCSV(outP.resolve("out.csv"), iterId, s);
-        extraOutputsForThibaud(outP, s);
+        CsvOutput.writeDailyStats(outP.resolve("out.csv"), iterId, s);
```

Comment:
We might want to push the "out.csv" decision into the CsvOutput class (since "deaths.csv" etc are set there so it's a single place if we ever want to change it) - very minor thing so happy to leave this for later.